util/site/site-builder/builder.Dockerfile

```diff
@@ -18,7 +18,27 @@ ENV LANGUAGE en_US:en
 
 RUN mkdir -p /tools
 
+##############
+### nodejs ###
+##############
+
+# Install nodejs
+ENV NVM_DIR "/tools/.nvm"
+ENV PATH "/tools/.nvm/versions/node/v18.7.0/bin:${PATH}"
+RUN mkdir /tools/.nvm \
+    && curl -so- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash \
+    && . "/tools/.nvm/nvm.sh" \
+    && nvm install v18.7.0
+
+# Install chrome-unstable for puppeteer
+RUN curl -so/tmp/chrome.deb https://dl.google.com/linux/direct/google-chrome-unstable_current_amd64.deb \
```

Comment:
We can't pin this to a pre-release version yet, right? *Unstable* by definition sounds like we can expect problems in the longer term

Reply:
@jprendes Thoughts on this?

Reply:
We (for the diagram) don't need any "unstable" feature, but this is what upstream `puppeteer` recommends.
I assume that's what they test against :-)

Reply:
This would go away if we merge https://github.com/lowRISC/opentitan/pull/17552, right? If not or we don't proceed with https://github.com/lowRISC/opentitan/pull/17552, we should create an issue to get away from unstable.

Reply:
> @hcallahan-lowrisc: Please don't forget to create an issue to track this (and the [item below](https://github.com/lowRISC/opentitan/pull/17405#discussion_r1137329522))

https://github.com/lowRISC/opentitan/pull/17405#issuecomment-1467945506
It's on my todos :+1: